tables/nlb_access_log/nlb_access_log_table.go

```diff
@@ -0,0 +1,82 @@
+package nlb_access_log
+
+import (
+	"strings"
+	"time"
+
+	"github.com/rs/xid"
+	"github.com/turbot/pipe-fittings/v2/utils"
+	"github.com/turbot/tailpipe-plugin-aws/sources/s3_bucket"
+	"github.com/turbot/tailpipe-plugin-sdk/artifact_source"
+	"github.com/turbot/tailpipe-plugin-sdk/artifact_source_config"
+	"github.com/turbot/tailpipe-plugin-sdk/constants"
+	"github.com/turbot/tailpipe-plugin-sdk/mappers"
+	"github.com/turbot/tailpipe-plugin-sdk/row_source"
+	"github.com/turbot/tailpipe-plugin-sdk/schema"
+	"github.com/turbot/tailpipe-plugin-sdk/table"
+)
+
+const NlbAccessLogTableIdentifier = "aws_nlb_access_log"
+
+const nlbLogFormat = `$type $version $timestamp $elb $listener $client $destination $connection_time $tls_handshake_time $received_bytes $sent_bytes $incoming_tls_alert $chosen_cert_arn $chosen_cert_serial $tls_cipher $tls_protocol_version $tls_named_group $domain_name $alpn_fe_protocol $alpn_be_protocol $alpn_client_preference_list $tls_connection_creation_time`
```

Comment:
`client` & `destination` could be preemptively split in the pattern (If we know 100% we will always get them and not get `-`, otherwise can ignore this.

Reply:
Additional note - `regex` mapper is probably better in terms of performance and capabilities but this works for fixed format